README.md

```diff
@@ -23,30 +24,47 @@ information stored in a Postgres database:
 
 ```mermaid
 sequenceDiagram
-    actor C as Coke User
-    actor P as Pepsi User
+    actor C1 as Coke User (Americas)
+    actor C2 as Coke User (Europe)
+    actor P as Pepsi User (Americas)
     participant M as Multitenant Persistence
     participant A as Authorizer
     participant DB as Postgres
 
     rect rgb(200, 225, 250)
-    C ->>+ M: Find VM1
+    C1 ->>+ M: Find VM1
+    M ->>+ A: GetOrgID
+    A -->>- M: Coke
+    M ->>+ A: GetInstanceID
```

Comment:
It's really an "instancer" that fetches instance ID rather than the authorizer. Would you like to add a figure for instancer to the diagram to reflect that?

Reply:
authorizer is the package here, at somepoint we need to separate it out into Tenancer and Instancer